src/App.tsx

```diff
@@ -1,38 +1,26 @@
 import * as React from 'react';
 import './App.css'
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
 
-const list = [
-  {
-    title: 'Cleaning',
-    id: 0,
-  }, {
-    title: 'Groceries',
-    id: 1,
-  }, {
-    title: 'Work',
-    id: 2,
-  },
-  {
-    title: 'Sleep',
-    id: 3,
-  },
-];
+// @Ethan I'm going to use sudo code to discuss questions/topics that come up while i'm coding.
+// I do understand that it is not a conventional way of communication or using sudo code
+// please let me know how to leave small questions for discuss as such.
 
-function App() {
-  return (
-    <div className="body-wrapper">
-      <ul className="list-wrapper">
-        {list.map((item) => {
-          return (
-            <li key={item.id}>
-              <input type="checkbox" id={item.title} name={item.title} value={item.title}/>
-              <label htmlFor={item.title}>{item.title}</label>
-            </li>
-          )
-        })}
-      </ul>
-    </div>
-  )
-}
+// [TO THINK ABOUT] 
+// @Ethan functional declaration cs functional expression
```

Comment:
If you need to call a function before its declaration, you might prefer a function declaration for hoisting to take effect. Function expressions are a common practice in React. That approach helps to keep the component definition concise and encapsulated within the scope of the constant (in this case App).